Oluwafayokemi/Maintenance-Tracker-App
# Ch set up server install packages #157674944

**what does this pr do?**
set server side and upload the packages needed for the server to run successfully
**Description of Task to be completed**
- install all relevant packages
- create app.js file and run routes
**How should this be manually tested?**
- git clone this repository or git pull
- on your terminal npm install to install packages to your folder
- on your terminal run npm start or npm run start:dev 
you should see this 
![image](https://user-images.githubusercontent.com/33626790/40199696-2e8b7214-5a12-11e8-8bd2-091ec6267d99.png)

**what are the relevant pivotal tracker stories?**
#6
